Daymorelah/RideMyWay
# Refactor table relationship

#### What does this PR do?
This chore refactors the implementation of the table relationship used in previous branches.
#### Description of Task to be completed?
* Delete irrelevant tables previously created
* Create new tables and define their relationships
#### How should this be manually tested?
* Clone the repo, `cd` into the  project
* Ensure you are in the test environment
* Run `npm run models` then run `npm test`
